### ISSUE TITLE
Sharing options: Student code - need to simplify the wording in the dialog

### ISSUE THREAD

**Issue by corinnewo** [ji-devs/ji-cloud#3763]
Teachers are finding this hard to understand:

1. Once code generated, don't need the button anymore
2. switch the order - put code first and then the URL
3. @ChaykeeMor to add text to explain where to go to input the code
4. @GaliShapira to add mockup here

![image](https://user-images.githubusercontent.com/16290361/225955652-a5c1ee0b-75d5-44ec-bd72-820f68f54073.png)

**Comment by ChaykeeMor** [comment]
@GaliShapira 

Where? | What? | Change to:
-- | -- | --
Title | Share with Students  |  Share with students
Button | Click to create a code: | Click to create a code 
Above URL | Ask the students to go to: | OR go straight to link:
Link | Copy URL | Copy link
After Click | URL copied | Link copied
Above code | Student code: | Go to Jigzi.org/kids and input this code:
Link | Copy Code | NO CHANGE
After Click  | Student code copied | Code copied

NEW ORDER: 
Share with students
Click to create a code
Go to Jigzi.org/kids and input this code:
CODE
OR go straight to link:
LINK

**Comment by ChaykeeMor** [comment]
Closing in favor of #3988
